src/js/helpers/graph/declarationHelper.mock.js

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const singleNode = {
+  id: 'rootId', label: 'A', status: 'success', type: 'single',
+};
+
+export const compositeNode = {
+  id: 'rootId',
+  label: 'Test',
+  status: 'error',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'sub1', label: 's1', status: 'success', type: 'single',
+    },
+    {
+      id: 'sub2', label: 's2', status: 'success', type: 'single',
+    },
+  ],
+};
+
+export const compositeWithTransitions = {
```

Comment:
It is a complex scenario, please start with a simple one:
```
export const compositeNodeWithTransitions = {
  id: 'rootId',
  label: 'Test',
  status: 'success',
  type: 'composite',
  subtasks: [
    {
      id: 'sub1', label: 's1', status: 'success', type: 'single', response: { ... }
    },
    {
      id: 'sub2', label: 's2', status: 'success', type: 'single', response: { ... }
    },
  ],
  response: {
    transition: '_success',
  },
  on: {
    _success: {
      id: 'last',
      label: 'Test2',
      status: 'success',
      type: 'single',
      response: { ... }
    },
  }
};
```
and validate only transitions. Then you can introduce your more complex scenario.